.circleci/config.yml

```diff
@@ -985,6 +1022,18 @@ workflows:
                         ignore:
                             - staging
                             - master
+            - lint:
```

Comment:
Should we run both linting and code_formatting on MERGE_TO_STAGING too?

Reply:
sure why not , should be align with unit test